api/src/main/resources/application.properties

```diff
@@ -1,8 +1,17 @@
 server.port=${port:8080}
 
+# when setting up local mysql for first time, use schema file /api/database/schema_create.sql
+
+# example for local mysql
+# spring.datasource.url=jdbc:mysql://localhost:3306/protectplayanow2
+# spring.datasource.username=username
+# spring.datasource.password=yourpassword
+
 spring.datasource.url=#DO NOT COMMIT THESE VALUES TO GITHUB
```

Comment:
Why not commit the data source URL for localhost with the same DB as in `schema_create.sql`, and only let the user set the username and password?

Reply:
good thinking, I'll change.